### DISCUSSION TITLE
Xbox version

### DISCUSSION THREAD

**Discussion by danstis** [axodox/unpaint#24] [Q&A] [answered]
Hi, I see the new release with the Xbox compatible version, how do we load this onto the xbox?

**Comment by axodox** [comment]
Sorry, that was my mistake, the build script accidentally published it as release.

You would need to have [dev mode activated](https://www.howtogeek.com/703443/how-to-put-your-xbox-series-x-or-s-into-developer-mode/) on it for now. But later, if MS is willing, it could be in the store. 

However for now the UI is barely usable on the Xbox, and there are several other issues to solve. I made it generate some images with quick and dirty hacks yesterday, and it looked ok. It will need a few days of work to get it in order. I will let you know when it is ready.

**Comment by axodox** [comment, marked as answer]
Here you go: [Unpaint demonstration on Xbox consoles](https://github.com/axodox/unpaint/wiki/Running-Unpaint-on-the-Xbox-Series-consoles), see it on [YouTube](https://www.youtube.com/watch?v=CWhoRcQttCM)
